ci-operator/step-registry/hypershift/hostedcluster/create/hostedcluster/hypershift-hostedcluster-create-hostedcluster-ref.yaml

```diff
@@ -38,6 +41,36 @@ ref:
   - name: HYPERSHIFT_INFRA_AVAILABILITY_POLICY
     default: "SingleReplica"
     documentation: "Availability policy for infrastructure services in guest cluster. Supported options: SingleReplica, HighlyAvailable, default SingleReplica"
+  - name: POWERVS_REGION
+    default: ""
+    documentation: "The PowerVS region of the cluster."
+  - name: POWERVS_ZONE
+    default: ""
+    documentation: "The PowerVS zone of the cluster."
+  - name: POWERVS_VPC_REGION
+    default: ""
+    documentation: "The region of the VPC created for PowerVS deployment"
+  - name: POWERVS_RESOURCE_GROUP
+    default: ""
+    documentation: "The PowerVS resource group."
+  - name: POWERVS_PROC_TYPE
+    default: ""
+    documentation: "The processor type of Power nodes."
+  - name: POWERVS_PROCESSORS
+    default: ""
+    documentation: "The processor count of Power nodes."
+  - name: POWERVS_SYS_TYPE
+    default: ""
+    documentation: "The system type of Power nodes."
+  - name: POWERVS_GUID
+    default: ""
+    documentation: "IBM Cloud PowerVS Service Instance ID.This flag is used to reuse an existing PowerVS Service Instance resource for cluster's infra"
+  - name: VPC
+    default: ""
+    documentation: "IBM Cloud VPC Name. This flag is used to reuse an existing VPC resource for cluster's infra"
+  - name: CLOUD_CONNECTION
+    default: ""
+    documentation: "Cloud Connection in given zone.This flag is used to reuse an existing Cloud Connection resource for cluster's infra"
```

Comment:
it will be good to prefix all the environment variable with `POWERVS` to easily identify for the platform

Reply:
Done